catkin_ws/src/planner/src/mission.py

```diff
@@ -68,6 +69,22 @@ def Tricks(t):
             res = "trick not identified"
     endMission("Finished trick. Result {}".format(res))
 
+
+def octogone_task():
+    sm = smach.StateMachine(outcomes=['success', 'failure'])
+    ocotogne_target_class = 3 
+    with sm:
+        smach.StateMachine.add('find_octogone', BreadthFirstSearch(timeout=9999, target_classes=[(ocotogne_target_class, 1)], control=control, mapping=mapping), 
```

Comment:
We should move the AUV to the octagon approximate position before breadth first search

Reply:
ideally should also support having several octagon position and at each failure we try the next one